src/osx/Installer.Mac/pack.sh

```diff
@@ -11,9 +11,16 @@ SRC="$ROOT/src"
 OUT="$ROOT/out"
 INSTALLER_SRC="$SRC/osx/Installer.Mac"
 
+if [ ! -z "$HOMEBREW_PREFIX" ]; then
+    echo "Running in Homebrew environment, installing to Homebrew-recognized path..."
+    PREFIX="$HOMEBREW_PREFIX"
+else
+    PREFIX="/usr/local"
+fi
```

Comment:
`pack.sh` isn't going to be run in a Homebrew environment. We need to switch and hardcode the default prefixes for the different target runtimes:

Runtime|`$PREFIX`
-|-
`osx-x64`|`/usr/local`
`osx-arm64`|`/opt/homebrew`

We should add a `--runtime` argument to this script and pass the correct RID through from `build.sh` as well as from `release.yml`.

```shell
# Parse script arguments
for i in "$@"
do
...
    --runtime=*)
    RUNTIME="${i#*=}"
    shift # past argument=value
    ;;
...
```

```shell
if [ -z "$RUNTIME" ]; then
    die "--runtime was not set"
fi

case "$RUNTIME" in
    "osx-x64")
        PREFIX="/usr/local"
        ;;
    "osx-arm64")
        PREFIX="/opt/homebrew"
        ;;
    *)
        die "Unknown runtime '$RUNTIME'"
        ;;
esac

...

INSTALL_LOCATION="$PREFIX/share/gcm-core"
```

Reply:
I should also mention that the `conclusion.html` resource file that gets included in the installer package also references `/usr/local` so that will need to be parameterised/swapped like we do for the `distribution.$ARCH.xml` file.

Reply:
Gotcha. I think the issue I have with this is that we're always going to install `osx-arm64` to `/opt/homebrew`, right? Is this desired behavior when a user is installing the `.pkg` directly from the releases page, for example?

Reply:
Resolved offline!